services/payments/src/simcore_service_payments/services/notifier_email.py

```diff
@@ -0,0 +1,264 @@
+    <p>For more details you can view or download your <a href="{{ payment.invoice_url }}">receipt</a></p>
+    <p>Should you have any questions or require further assistance, please do not hesitate to reach out to our <a href="mailto:{{ product.support_email }}">customer support team</a>.</p>
+    <p>Best Regards,</p>
+    <p>{{ product.display_name }} support team<br>{{ product.vendor_display_inline }}</p>
+</div>
+{% endblock %}
+"""
+
+_NOTIFY_PAYMENTS_TXT = """
+    Dear {{ user.first_name }},
+
+    We are delighted to confirm the successful processing of your payment of **{{ payment.price_dollars }}** *USD* for the purchase of **{{ payment.osparc_credits }}** *credits*. The credits have been added to your {{ product.display_name }} account, and you are all set to utilize them.
+
+    To view or download your detailed receipt, please click the following link {{ payment.invoice_url }}
+
+    Should you have any questions or require further assistance, please do not hesitate to reach out to our {{ product.support_email }}" customer support team.
+    Best Regards,
+
+    {{ product.display_name }} support team
+    {{ product.vendor_display_inline }}
+"""
+
+
+_NOTIFY_PAYMENTS_SUBJECT = "Your Payment {{ payment.price_dollars }} USD for {{ payment.osparc_credits }} Credits Was Successful"
+
+
+_PRODUCT_NOTIFICATIONS_TEMPLATES = {
+    "base.html": _BASE_HTML,
+    "notify_payments.html": _NOTIFY_PAYMENTS_HTML,
+    "notify_payments.txt": _NOTIFY_PAYMENTS_TXT,
+    "notify_payments-subject.txt": _NOTIFY_PAYMENTS_SUBJECT,
+}
+
+
+@dataclass
+class _UserData:
+    first_name: str
+    last_name: str
+    email: str
+
+
+@dataclass
+class _ProductData:
+    product_name: ProductName
+    display_name: str
+    vendor_display_inline: str
+    support_email: str
+
+
+@dataclass
+class _PaymentData:
+    price_dollars: str
+    osparc_credits: str
+    invoice_url: str
+
+
+async def _create_user_email(
+    env: Environment,
+    user: _UserData,
+    payment: _PaymentData,
+    product: _ProductData,
+) -> EmailMessage:
+    # data to interpolate template
+    data = {
+        "user": user,
+        "product": product,
+        "payment": payment,
+    }
+
+    msg = EmailMessage()
+
+    msg["From"] = Address(
+        display_name=f"{product.display_name} support",
+        addr_spec=product.support_email,
+    )
+    msg["To"] = Address(
+        display_name=f"{user.first_name} {user.last_name}",
+        addr_spec=user.email,
+    )
+    msg["Subject"] = env.get_template("notify_payments-subject.txt").render(data)
+
+    # Body
+    text_template = env.get_template("notify_payments.txt")
+    msg.set_content(text_template.render(data))
+
+    html_template = env.get_template("notify_payments.html")
+    msg.add_alternative(html_template.render(data), subtype="html")
+    return msg
+
+
+def _guess_file_type(file_path: Path) -> tuple[str, str]:
+    assert file_path.is_file()
+    mimetype, _encoding = mimetypes.guess_type(file_path)
```

Comment:
Just as an info

You might also be interested in https://pypi.org/project/python-magic/
we use it for the `dynamic-sidecar`. Not sure if it does the same or is any better/worse. It's an alternative.